ERROR - 2024-12-08 05:17:45 --> Error connecting to the database: mysqli_sql_exception: Connection refused in /Applications/XAMPP/xamppfiles/htdocs/framework/system/Database/MySQLi/Connection.php:187
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/framework/system/Database/MySQLi/Connection.php(187): mysqli->real_connect('127.0.0.1', 'root', '', 'blog_db', 3306, '', 0)
#1 /Applications/XAMPP/xamppfiles/htdocs/framework/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /Applications/XAMPP/xamppfiles/htdocs/framework/system/Database/BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 /Applications/XAMPP/xamppfiles/htdocs/framework/system/Database/BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 /Applications/XAMPP/xamppfiles/htdocs/framework/system/Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 /Applications/XAMPP/xamppfiles/htdocs/framework/system/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 /Applications/XAMPP/xamppfiles/htdocs/framework/app/Controllers/Blog.php(17): CodeIgniter\BaseModel->findAll()
#7 /Applications/XAMPP/xamppfiles/htdocs/framework/system/CodeIgniter.php(933): App\Controllers\Blog->index()
#8 /Applications/XAMPP/xamppfiles/htdocs/framework/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Blog))
#9 /Applications/XAMPP/xamppfiles/htdocs/framework/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 /Applications/XAMPP/xamppfiles/htdocs/framework/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 /Applications/XAMPP/xamppfiles/htdocs/framework/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 /Applications/XAMPP/xamppfiles/htdocs/framework/public/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 /Applications/XAMPP/xamppfiles/htdocs/framework/system/rewrite.php(44): require_once('/Applications/X...')
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Connection refused in /Applications/XAMPP/xamppfiles/htdocs/framework/system/Database/MySQLi/Connection.php:231
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/framework/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /Applications/XAMPP/xamppfiles/htdocs/framework/system/Database/BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 /Applications/XAMPP/xamppfiles/htdocs/framework/system/Database/BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 /Applications/XAMPP/xamppfiles/htdocs/framework/system/Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#4 /Applications/XAMPP/xamppfiles/htdocs/framework/system/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#5 /Applications/XAMPP/xamppfiles/htdocs/framework/app/Controllers/Blog.php(17): CodeIgniter\BaseModel->findAll()
#6 /Applications/XAMPP/xamppfiles/htdocs/framework/system/CodeIgniter.php(933): App\Controllers\Blog->index()
#7 /Applications/XAMPP/xamppfiles/htdocs/framework/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Blog))
#8 /Applications/XAMPP/xamppfiles/htdocs/framework/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 /Applications/XAMPP/xamppfiles/htdocs/framework/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 /Applications/XAMPP/xamppfiles/htdocs/framework/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 /Applications/XAMPP/xamppfiles/htdocs/framework/public/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 /Applications/XAMPP/xamppfiles/htdocs/framework/system/rewrite.php(44): require_once('/Applications/X...')
#13 {main}
CRITICAL - 2024-12-08 05:17:45 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Connection refused
[Method: GET, Route: blog]
in SYSTEMPATH/Database/BaseConnection.php on line 467.
 1 SYSTEMPATH/Database/BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH/Database/BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `post`', [], false)
 3 SYSTEMPATH/Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH/Controllers/Blog.php(17): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Blog->index()
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Blog))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH/rewrite.php(44): require_once('/Applications/XAMPP/xamppfiles/htdocs/framework/public/index.php')
ERROR - 2024-12-08 05:17:53 --> Error connecting to the database: mysqli_sql_exception: Connection refused in /Applications/XAMPP/xamppfiles/htdocs/framework/system/Database/MySQLi/Connection.php:187
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/framework/system/Database/MySQLi/Connection.php(187): mysqli->real_connect('127.0.0.1', 'root', '', 'blog_db', 3306, '', 0)
#1 /Applications/XAMPP/xamppfiles/htdocs/framework/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /Applications/XAMPP/xamppfiles/htdocs/framework/system/Database/BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 /Applications/XAMPP/xamppfiles/htdocs/framework/system/Database/BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 /Applications/XAMPP/xamppfiles/htdocs/framework/system/Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 /Applications/XAMPP/xamppfiles/htdocs/framework/system/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 /Applications/XAMPP/xamppfiles/htdocs/framework/app/Models/CategoryModel.php(12): CodeIgniter\BaseModel->findAll()
#7 /Applications/XAMPP/xamppfiles/htdocs/framework/app/Controllers/Category.php(14): App\Models\CategoryModel->getAllCategories()
#8 /Applications/XAMPP/xamppfiles/htdocs/framework/system/CodeIgniter.php(933): App\Controllers\Category->index()
#9 /Applications/XAMPP/xamppfiles/htdocs/framework/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Category))
#10 /Applications/XAMPP/xamppfiles/htdocs/framework/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 /Applications/XAMPP/xamppfiles/htdocs/framework/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 /Applications/XAMPP/xamppfiles/htdocs/framework/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 /Applications/XAMPP/xamppfiles/htdocs/framework/public/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 /Applications/XAMPP/xamppfiles/htdocs/framework/system/rewrite.php(44): require_once('/Applications/X...')
#15 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Connection refused in /Applications/XAMPP/xamppfiles/htdocs/framework/system/Database/MySQLi/Connection.php:231
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/framework/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /Applications/XAMPP/xamppfiles/htdocs/framework/system/Database/BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 /Applications/XAMPP/xamppfiles/htdocs/framework/system/Database/BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 /Applications/XAMPP/xamppfiles/htdocs/framework/system/Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#4 /Applications/XAMPP/xamppfiles/htdocs/framework/system/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#5 /Applications/XAMPP/xamppfiles/htdocs/framework/app/Models/CategoryModel.php(12): CodeIgniter\BaseModel->findAll()
#6 /Applications/XAMPP/xamppfiles/htdocs/framework/app/Controllers/Category.php(14): App\Models\CategoryModel->getAllCategories()
#7 /Applications/XAMPP/xamppfiles/htdocs/framework/system/CodeIgniter.php(933): App\Controllers\Category->index()
#8 /Applications/XAMPP/xamppfiles/htdocs/framework/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Category))
#9 /Applications/XAMPP/xamppfiles/htdocs/framework/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 /Applications/XAMPP/xamppfiles/htdocs/framework/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 /Applications/XAMPP/xamppfiles/htdocs/framework/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 /Applications/XAMPP/xamppfiles/htdocs/framework/public/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 /Applications/XAMPP/xamppfiles/htdocs/framework/system/rewrite.php(44): require_once('/Applications/X...')
#14 {main}
CRITICAL - 2024-12-08 05:17:53 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Connection refused
[Method: GET, Route: category]
in SYSTEMPATH/Database/BaseConnection.php on line 467.
 1 SYSTEMPATH/Database/BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH/Database/BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `category`', [], false)
 3 SYSTEMPATH/Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH/Models/CategoryModel.php(12): CodeIgniter\BaseModel->findAll()
 6 APPPATH/Controllers/Category.php(14): App\Models\CategoryModel->getAllCategories()
 7 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Category->index()
 8 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Category))
 9 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH/rewrite.php(44): require_once('/Applications/XAMPP/xamppfiles/htdocs/framework/public/index.php')
